code/track.ss

```diff
@@ -40,7 +42,7 @@
 
 ;;; UUID
 
-;; wrapper to read uuid generated by configlet from scheme
+;; wrapper to read uuid generated by bonfiglet from scheme
```

Comment:
?

Reply:
🤣 sorry for the typo.  I just rewrite it when it was moved.

I will fix it.

Reply:
Fixed